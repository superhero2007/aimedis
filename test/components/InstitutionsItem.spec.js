import React from 'react';
import { mount } from 'enzyme';

import InstitutionsItem from 'components/InstitutionsItem';

function setup(align, description = 'Description', src = 'institutions-bg1.png', subtitle = 'SubTitle', title = 'Title') {
  return mount(<InstitutionsItem
    align={align}
    description={description}
    src={src}
    subtitle={subtitle}
    title={title}
  />);
}

describe('InstitutionsItem', () => {
  const leftWrapper = setup('left');

  it('should be a StatelessComponent', () => {
    expect(leftWrapper.instance()).toBeNull();
  });

  it('should render properly', () => {
    expect(leftWrapper.html()).toMatchSnapshot();
  });

  const rightWrapper = setup('right');

  it('should render properly', () => {
    expect(rightWrapper.html()).toMatchSnapshot();
  });
});
