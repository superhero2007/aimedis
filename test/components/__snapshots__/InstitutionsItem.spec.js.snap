// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstitutionsItem should render properly 1`] = `"<div class=\\"app__institutions__item\\"><div class=\\"app__institutions__item__content\\"><div class=\\"row\\"><div class=\\"col-6 app__institutions__item__content__info\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">SubTitle</p><p class=\\"app__title__title\\">Title</p></div></div><p class=\\"app__institutions__item__content__info__description\\">Description</p><button class=\\"app__institutions__item__content__info__download btn btn__secondary\\" type=\\"button\\">DOWNLOAD THE CASE STUDY (8.4 MB)</button></div></div></div><div class=\\"col-6 d-none d-lg-flex app__institutions__item__content__image\\"><div class=\\"app__institutions__item__content__image__content\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><img src=\\"\\" alt=\\"No Asset\\"></div></div></div></div></div></div>"`;

exports[`InstitutionsItem should render properly 2`] = `"<div class=\\"app__institutions__item\\"><div class=\\"app__institutions__item__content\\"><div class=\\"row\\"><div class=\\"col-6 d-none d-lg-flex direction-rtl app__institutions__item__content__image\\"><div class=\\"app__institutions__item__content__image__content\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><img src=\\"\\" alt=\\"No Asset\\"></div></div></div><div class=\\"col-6 app__institutions__item__content__info\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">SubTitle</p><p class=\\"app__title__title\\">Title</p></div></div><p class=\\"app__institutions__item__content__info__description\\">Description</p><button class=\\"app__institutions__item__content__info__download btn btn__secondary\\" type=\\"button\\">DOWNLOAD THE CASE STUDY (8.4 MB)</button></div></div></div></div></div></div>"`;
